cmd/brew-services.rb

```diff
@@ -173,7 +173,8 @@ def run!
       homebrew!
       usage if ARGV.empty? || ARGV.include?("help") || ARGV.include?("--help") || ARGV.include?("-h")
 
-      odie "brew services cannot run under tmux!" if ENV["TMUX"]
+      # pbpaste's exit status is a proxy for detecting the use of reattach-to-user-namespace
+      odie "brew services cannot run under tmux!" if ENV["TMUX"] && !system("/usr/bin/pbpaste", "&>/dev/null")
```

Comment:
Maybe use the block form of `if` to make this more readable/less long a line. Also, using `"&>/dev/null"` as a separate argument won't do what you think it will; it'll pass that as an argument. Just remove embed the space in there as there's no user-submitted input we're worried about here.